pkg/converter/convert_unix.go

```diff
@@ -37,7 +37,7 @@ const envNydusWorkdir = "NYDUS_WORKDIR"
 func getBuilder() string {
 	builderPath := os.Getenv(envNydusBuilder)
 	if builderPath == "" {
-		builderPath = "nydus-image"
+		builderPath = "/root/project/image-service/target-fusedev/release/nydus-image"
```

Comment:
it'd be better to drop the immediate temporary modification?

Reply:
It's been fixed and what you seem to see is stale code

Reply:
Frankly speaking, some of the small projects I am involved in general only require a detailed description in the PR, and multiple commits in the PR are more casual and do not need to be merged and sorted out, and the commit message does not need to be so detailed.